tensorflow/compiler/xla/service/gpu/custom_call_test.cc

```diff
@@ -13,28 +13,54 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+#if GOOGLE_USE_CUDA
 #include "third_party/gpus/cuda/include/cuda.h"
 #include "third_party/gpus/cuda/include/cuda_runtime_api.h"
 #include "third_party/gpus/cuda/includes/cuda_headers/third_party/gpus/cuda/include/driver_types.h"
+#define PLATFORM "CUDA"
+#elif TENSORFLOW_USE_ROCM
+#include "rocm/include/hip/hip_runtime.h"
+#define PLATFORM "ROCM"
+#endif
 #include "tensorflow/compiler/xla/client/lib/constants.h"
 #include "tensorflow/compiler/xla/client/xla_builder.h"
 #include "tensorflow/compiler/xla/service/custom_call_target_registry.h"
 #include "tensorflow/compiler/xla/status_macros.h"
 #include "tensorflow/compiler/xla/test_helpers.h"
 #include "tensorflow/compiler/xla/tests/client_library_test_base.h"
 #include "tensorflow/core/lib/core/status_test_util.h"
+#include "tensorflow/stream_executor/gpu/gpu_types.h"
+
+
+#if GOOGLE_CUDA
+#define gpuSuccess cudaSuccess
```

Comment:
`gpuSuccess` is actually defined in `tensorflow/core/util/gpu_kernel_helper.h` already.

for the rest of the macros perhaps it's fine to leave them here as they are only used by this test case.

Reply:
tensorflow/core/util/gpu_kernel_helper.h  brings in quite a few symbols and dependencies. So not sure if its worth it....

Reply:
got it. I'm fine leaving it here.